### PR TITLE
Gracefully handle containers removed from c/storage

### PR DESCRIPTION
**Pull request by mheon**
Allow containers that no longer exist in storage to be evicted from the state instead of erroring.

### CODE REVIEW

File: libpod/container_internal.go [hunk 1]
```diff
@@ -791,6 +791,15 @@ func (c *Container) cleanupStorage() error {
 
 	// Also unmount storage
 	if err := c.runtime.storageService.UnmountContainerImage(c.ID()); err != nil {
+		// If the container has already been removed, warn but don't
+		// error
+		// We still want to be able to kick the container out of the
+		// state
+		if err == storage.ErrNotAContainer || err == storage.ErrContainerUnknown {
+			logrus.Errorf("Storage for container %s has been removed", c.ID())
```
[the remark this answers is not in the document]
Comment (mheon) [reply libpod/container_internal.go]: cleanupStorage triggers a lot more often than just removing a container - after a container is stopped, for example. I think it's error-worthy to tell people the storage for a container is gone in that sort of case.

**Comment by mheon** [comment]
F27 test is a network flake

**Comment by mheon** [comment]
bot, retest this please

**Comment by mheon** [comment]
@rhatdan @baude PTAL

**Comment by rhatdan** [comment]
LGTM
@rh-atomic-bot r+

**Pull request closed without merging**